geth/jail/jail_test.go

```diff
@@ -128,7 +129,54 @@ func (s *JailTestSuite) TestPublicCreateAndInitCell() {
 
 func (s *JailTestSuite) TestPublicCreateAndInitCellWithJS() {
 	response := s.Jail.CreateAndInitCell("cell1", "var a = 2", "a")
-	s.Equal(`{"result": 2}`, response)
+	s.Equal(`{"result":2}`, response)
+}
+
+func (s *JailTestSuite) TestPublicCreateAndInitCellWithJSAndStringOutput() {
```

Comment:
You can also put all these test cases in one test function and use table-driven approach: https://github.com/golang/go/wiki/TableDrivenTests (many examples within status-go codebase as well).

Reply:
+1, @adriacidre please ping me when you fix that. Otherwise, it looks great!

Reply:
@mandrigin all fixed! I think CI is broken on develop, so not much can do to get it green :/